Apps/Medication/src/IMedicationDBContextExt.cs

```diff
@@ -0,0 +1,31 @@
+//-------------------------------------------------------------------------
```

Comment:
Lets put all the DB files into a single subfolder under src

Reply:
Done